source/fe/fe_abf.cc

```diff
@@ -187,7 +187,7 @@ FE_ABF<dim>::initialize_support_points(const unsigned int deg)
         poly[d].push_back(Polynomials::Monomial<double>(deg + 1));
       poly[dd] = Polynomials::Monomial<double>::generate_complete_basis(deg);
 
-      polynomials_abf[dd] = new AnisotropicPolynomials<dim>(poly);
+      polynomials_abf[dd].reset(new AnisotropicPolynomials<dim>(poly));
```

Comment:
The usual idiom would be
```suggestion
      polynomials_abf[dd] = std::make_unique<AnisotropicPolynomials<dim>>(poly);
```
so that you can also avoid the explicit call to `new`.